src/crimson/os/seastore/onode_manager/staged-fltree/node.cc

```diff
@@ -113,13 +128,27 @@ bool tree_cursor_t::Cache::validate_is_latest(const LeafNode& node,
     return false;
   if (!is_latest())
     return false;
-  auto [_key_view, _p_value] = node.get_kv_versioned(pos);
+  auto [_key_view, _p_value_header] = node.get_kv_versioned(pos);
   if (*key_view != _key_view)
     return false;
-  if (p_value != _p_value)
+  if (p_value_header != _p_value_header)
     return false;
   return true;
 }
+
+std::pair<NodeExtentMutable&, ValueDeltaRecorder*>
+tree_cursor_t::Cache::prepare_mutate_value_payload(context_t c) {
```

Comment:
nit, i'd suggest move `{` to a new line in future when starting a function body in `.cc` file. so we can have a visual separator between the function signature and its function body.

Reply:
see https://github.com/ceph/ceph/pull/38924/commits/e86fe04e9b9a3f4e3488e11cdcc5ea8c3eaeb0c8